services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusService.kt

```diff
@@ -63,6 +67,22 @@ class HakemusService(
     private val paatosService: PaatosService,
 ) {
 
+    // @EventListener(ApplicationReadyEvent::class)
+    @Transactional
+    fun testPdf() {
+        logger.info { "Finding application with hardcoded id..." }
+        val hakemus = getById(14L)
+
+        logger.info { "Creating PDF..." }
+        val attachment = getApplicationDataAsPdf(hakemus.id, "HAI24-12", hakemus.applicationData)
+
+        logger.info { "Writing PDF to file ${attachment.metadata.name}..." }
+        File(attachment.metadata.name).writeBytes(attachment.file)
+
+        logger.info { "PDF test complete." }
+        exitProcess(0)
+    }
+
```

Comment:
This needs to be removed before merging. I used this to test the PDF layout and left it here for the reviewer.